nic_api/models.py

```diff
@@ -263,7 +267,10 @@ def from_xml(cls, rr):
         id_ = rr.attrib['id'] if 'id' in rr.attrib else None
         name = rr.find('name').text
         idn_name = rr.find('idn-name').text
-        ttl = rr.find('ttl').text
+	if rr.find('ttl'):
```

Comment:
Sometimes `<ttl></ttl>` section was absent in XML record representations I got from server, so to be accurate, `ttl` field should be optional both in class, in from_xml() and in to_xml() methods.
I have implementation for this, but I did not tested it yet because of lack of time to work on this project :(
I will be happy to see your pull request fixed, if you want to contribute!

Reply:
You are satisfied with this verification implementation? If yes, I could add this on all functions. Perhaps it would have looked better as a decorator/function?

Reply:
I just tested and pushed fix for TTL fields, thanks Vadim. You can work on non-implemented `to_rr()` methods, the format of XML representations can be found in this document: https://www.nic.ru/help/upload/file/API_DNS-hosting.pdf